src/store/sagas.js

```diff
@@ -151,5 +151,5 @@ export default function* rootSaga() {
   if (NATIVE_MOBILE) {
     sagas.push(initInterface)
   }
-  yield sagas.map(fork)
+  yield all(sagas.map(fork))
```

Comment:
wonder if we actually need the fork

Reply:
just checked, looks like `all` doesn't fork automatically :(